<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnled_On">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnled_Off">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_led">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_buzzer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>86</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buzzer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnbuzz1">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>119</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnbuzz2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>118</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_distSensing">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>164</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Distance Sensing</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbldistance">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance : cm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btndSens1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btndSens2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
